pysumma/simulation.py

```diff
@@ -2,9 +2,11 @@
 import copy
 import shutil
 import subprocess
+import numpy as np
```

Comment:
Is there a reason to add NumPy and Iterable library? This library doesn't use in simulation,py